cli/daemon/dash/trace.go

```diff
@@ -323,9 +323,7 @@ func (tp *traceParser) parseReq(req *tracepb.Request) (*Request, error) {
 		MessageID:      req.MessageId,
 		Attempt:        req.Attempt,
 		Published:      nil,
-
-		CallLoc: nullInt32(req.CallLoc),
-		DefLoc:  req.DefLoc,
+		DefLoc:         req.DefLoc,
```

Comment:
Even though we don't use in on the hosted version, the local dashboard still replies on this for now.